language/break_spec.rb

```diff
@@ -60,6 +60,13 @@
       ScratchPad.recorded.should == [:a, :za, :xa, :zd, :aa, :zb]
     end
   end
+
+  describe "from another thread" do
+    it "raises a LocalJumpError when getting the value from another thread" do
+      lambda { @program.break_in_another_thread }.should raise_error(LocalJumpError)
+      ScratchPad.recorded.should == [:a, :b]
```

Comment:
Improved in f0a147b389f93192963b0d6d631e29f7e71ac588, although I could not find a simpler way to test this --- I couldn't find any method that would wait for a thread to finish without actually raising the exception too, so I had to basically ignore it on the `join`.

The alternative would be to use a semaphore or a barrier, but that would be even more code and bigger guns for the test...

Any ideas on how to make this simpler?